src/shared/components/palette/case-link/services/linked-cases.service.ts

```diff
@@ -4,8 +4,12 @@ import { CaseLink } from '../domain';
 
 @Injectable()
 export class LinkedCasesService {
+  public caseFieldValue = [];
+  public isLinkedCasesEventTrigger = false;
   public caseId: string;
   public linkCaseReasons: LovRefDataModel[] = [];
   public linkedCases: CaseLink[] = [];
   public preLinkedCases: CaseLink[] = [];
+  public removedLinkedCases: CaseLink[] = [];
+  public editMode = false;
 }
```

Comment:
Is this is a service? Seems like this is not a service but a data model.

Reply:
@rayliangatos is a service and its more of holding the data of different flow